frame/assets/Cargo.toml

```diff
@@ -29,12 +29,18 @@ features = ["derive"]
 package = "parity-scale-codec"
 version = "2.0.0"
 
+[dev-dependencies]
+governance-registry = { version = "0.0.1", path = "../governance-registry", default-features = false }
+pallet-balances = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.11" }
+orml-tokens = { git = "https://github.com/open-web3-stack/open-runtime-module-library", rev = "b89b1f98fcf0f5c341fbef1d12bfd869a2f2a9fe" }
```

Comment:
we switched to v0.9.12, need to use `rev = "ee37b76a7a8cf159e6c41851942758746318bb84"`

Reply:
another reason for not using branch:)